lang/de_DE.yml

```diff
@@ -1,10 +1,4 @@
 de_DE:
-  SilverStripe\UserForms\Model\EditableFormField:
```

Comment:
looks like this is making some unintentional removals?

Same with eo.yml and id.yml

Reply:
I'm not quite sure what's going on here. Some of the translations were under the wrong key. I've pushed the correct keys up and restored translations that were missing before pulling them back down.

Reply:
ok, though still seems like the translations are being deleted on this PR?

Reply:
They are not being deleted ... they are being moved to the correct key